LegPos msg
---
bool success
string message